app/src/main/java/com/example/finportfolio/data/repository/SettingStoreImpl.kt

```diff
@@ -1,35 +1,41 @@
 package com.example.finportfolio.data.repository
 
 import android.content.Context
-import android.content.SharedPreferences
+import androidx.datastore.preferences.core.edit
+import androidx.datastore.preferences.core.stringPreferencesKey
+import androidx.datastore.preferences.preferencesDataStore
 import com.example.finportfolio.domain.repository.SettingStore
 import dagger.hilt.android.qualifiers.ApplicationContext
 import javax.inject.Inject
 import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.map
 import kotlinx.coroutines.withContext
 
-class SettingStoreImpl @Inject
-constructor(@ApplicationContext context: Context) : SettingStore {
+class SettingStoreImpl @Inject constructor(@ApplicationContext private val context: Context) :
+    SettingStore {
```

Comment:
Тоже форматирование поехало
```suggestion
class SettingStoreImpl 
@Inject constructor(
    @ApplicationContext private val context: Context
) : SettingStore {
```